main.tf

```diff
@@ -30,7 +30,7 @@ resource "aws_db_instance" "default" {
   instance_class              = "${var.instance_class}"
   allocated_storage           = "${var.allocated_storage}"
   storage_encrypted           = "${var.storage_encrypted}"
-  vpc_security_group_ids      = "${aws_security_group.default.*.id}"
+  vpc_security_group_ids      = ["${join("", aws_security_group.default.*.id)}"]
```

Comment:
Why the join+splat here? This `vpc_security_group_ids` is expecting a list. `aws_security_group.default.*.id` returns a list.

Reply:
maybe `[]` would be enough to fix the issue #23 
just added `splat+join` for consistency